### PR TITLE
HTML/CSS & JavaScript capstone project

### PR DESCRIPTION
**Pull request by joel-kalema**
- [x] I should personalize the content of my page. Choose a topic that is different than the one in the original design.
- [x] I should follow these design guidelines, including:
          - Colors.
          -Typography: font face, size and weight.
          -Layout: composition and space between elements.
- [x] The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.
- [x] I may use a CSS framework (for example Bootstrap) for styling if you want to.
- [x] I should build only these 2 pages:
           -The home page.
            -The about page.
- [x] Each of these pages should have versions for 2 different screen sizes:
          -Mobile: up to 768px wide.
          -Desktop: 768px or wider.
          -Interactions
- [x] Dynamic page
          -The section "Featured speakers" should be created dynamically in JavaScript.
          -I should use a JavaScript variable with the data about the speakers and use it when the page loads to create the 
           HTML for this section dynamically.
- [x] Once I have finished the development of the project, I should record a video presenting the features of the project 
       I built. It is a video with a maximum length of 5 minutes. The content of the video should include:
           -A description of the project.
           -A demo of the project features (different pages and different screen sizes).
            -You should also highlight some interesting piece of code or something you built that you are very proud of.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+              </li>
+              <li>
+                <a href="#" class="toggle-close">Programme</a>
+              </li>
+              <li><a href="#" class="toggle-close">Join</a></li>
+              <li><a href="#" class="toggle-close">Sponsors</a></li>
+              <li><a href="#" class="toggle-close importent">News</a></li>
+            </ul>
+          </div>
+        </nav>
```
Comment (thecodechaser) [inline index.html:60]: - [ ] When I open the mobile menu and switch to the desktop screen, The desktop menu options are not appearing I have to refresh the page to get the desktop menu. Please make sure your desktop menu is visible and appears in all conditions. 

See the image for reference 
image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+        </div>
+        <a href="#">Join Causes Of Global Warming</a>
+      </section>
+      <section class="speakers">
+        <h3>Causes Of Global Warming</h3>
+        <div class="speaker"></div>
+        <a href="#" class="more">
+          <i class="fa fa-chevron-down expand-arrow" aria-hidden="true"></i>
+        </a>
+      </section>
```
Comment (thecodechaser) [inline index.html:119]: - [ ] You did a great job on the design but there is a little issue, You are supposed to stick to the [Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) template as much as possible. Your featured speaker's section is a little different from the design. Please add the album images behind the featured speaker's image and make sure it matches the design on the template.

### Your design

image

### Behance design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED 🔴🟡 🔴 

## Hi @joel-kalema👋 

### You are doing a marvelous job on the capstone project and this is one of the best projects I have seen so far!
I really like your presentation for the project 👍

There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

## Required Changes ♻️
- [ ]  In the presentation video you should show some piece of code that you are very proud of, Please make sure your presentation video includes that. (Please also include the presentation video in the PR)
- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: media.css [hunk 3]
```diff
@@ -0,0 +1,420 @@
...
+  .speakers h3 {
+    color: #272a31a9;
+    font-size: 30px;
+    padding: 40px 0;
+  }
+
+  .bihande {
+    margin-top: -15px;
+    margin-right: -55px;
+  }
```
Comment (Wes-Isaac) [inline media.css]: - [ ] Great job making the change the previous reviewer requested. However, there is a minor issue, the images move when the size of the screen is changing which is causing disarray. Please fix that.


### Your Implementation

image

File: index.html [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+              <li>
+                <a href="about.html" class="toggle-close importent01">About</a>
+              </li>
+              <li>
+                <a href="#" class="toggle-close">Programme</a>
+              </li>
+              <li><a href="#" class="toggle-close">Join</a></li>
+              <li><a href="#" class="toggle-close">Sponsors</a></li>
+              <li><a href="#" class="toggle-close importent">News</a></li>
+            </ul>
```
Comment (Wes-Isaac) [inline index.html]: - [ ] As the previous reviewer pointed out, please make sure in after switching from mobile screen to desktop your desktop menu is visible. Now in the mobile version when the mobile menu is opened and I switch to the desktop screen the desktop menu is not there. Please fix that.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
Hi @joel-kalema, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Highlights
- Linters are on point ✔
- Splendid work with the README file 👀
- Proper use of GitHub flow 👌
- Awesome design 🙌

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by thecodechaser — APPROVED** [review]
## STATUS: APPROVED 🟢

## Hi @joel-kalema👋, 

### You did a marvelous job on the capstone project and this is one of the best projects I have seen so far!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**